src/shmem_team.c

```diff
@@ -140,14 +185,15 @@ int shmem_internal_team_init(void)
     }
     shmem_internal_team_pool[SHMEM_TEAM_WORLD_INDEX] = &shmem_internal_team_world;
     shmem_internal_team_pool[SHMEM_TEAM_SHARED_INDEX] = &shmem_internal_team_shared;
+    shmem_internal_team_pool[SHMEM_TEAM_NODE_INDEX] = &shmem_internal_team_node;
 
     /* Allocate pSync pool, each with the maximum possible size requirement */
     /* Create two pSyncs per team for back-to-back collectives and one for barriers.
      * Array organization:
      *
-     * [ (world) (shared) (team 1) (team 2) ...  (world) (shared) (team 1) (team 2) ... ]
-     *  <----------- groups 1 & 2-------------->|<------------- group 3 ---------------->
-     *  <--- (bcast, collect, reduce, etc.) --->|<------ (barriers and syncs) ---------->
+     * [ (world) (shared) (host) (team 1) (team 2) ...  (world) (shared) (host) (team 1) (team 2) ... ]
```

Comment:
`node`?

Reply:
done in [c5d1bfe](https://github.com/Sandia-OpenSHMEM/SOS/pull/1102/commits/c5d1bfe72cc5b42404928954010748169e4fe68f)